src/bin/parser.js

```diff
@@ -244,109 +247,49 @@ export default class Harmonic {
                     pattern: ':language',
                     replacement: lang
                 }],
-                structure: getStructure(config.i18n.default, lang, config.posts_permalink)
+                structure: getStructure(config.i18n.default, lang, permalink)
             });
 
-            metadata.file = postspath + file;
+            metadata.file = filesPath + file;
             metadata.filename = filename;
-            metadata.link = postPath;
+            metadata.link = filePath;
             metadata.lang = lang;
             metadata.default_lang = config.i18n.default !== lang; // FIXME https://github.com/JSRocksHQ/harmonic/issues/169
 
-            const postHTMLFile = template
+            const contentHTMLFile = template
                 .render({
-                    post: {
+                    [filesType]: {
                         content: md.markdown(),
-                        metadata
+                        metadata: metadata
                     },
-                    config,
-                    lang
```

Comment:
@viniciusdacal can you rebase on your branch on master (or manually re-add `lang` here)?